pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -107,7 +104,12 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		return nil, err
 	}
 
-	admissionController := admission.NewFromPlugins(kubeClient, strings.Split(server.AdmissionControl, ","), server.AdmissionControlConfigFile)
```

Comment:
and maybe externalize the "ServiceAccount" plugin name as a constant in the admission plugin to catch changes or drift